modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -1315,29 +1315,36 @@ def test___setitem__mask():
     "data",
     [
         {},
-        pytest.param(
-            {"id": [], "max_speed": [], "health": []},
-            marks=pytest.mark.xfail(
-                reason="Throws an exception because generally assigning Series or other objects of length different from DataFrame does not work right now"
-            ),
-        ),
+        {"id": [], "max_speed": [], "health": []},
+        {"id": [1], "max_speed": [2], "health": [3]},
+        {"id": [4, 40, 400], "max_speed": [111, 222, 333], "health": [33, 22, 11]},
     ],
-    ids=["empty", "empty_columns"],
+    ids=["empty_frame", "empty_cols", "1_length_cols", "2_length_cols"],
 )
 @pytest.mark.parametrize(
     "value",
-    [np.array(["one", "two"]), [11, 22]],
```

Comment:
Why are we no longer testing `ndarray` case? And why were these values removed?

Reply:
it seems that there is no much difference between `list` and `ndarray`. So I removed this case from a test to prevent the combinatorial explosion of the amount of test parameters.

Reply:
speaking about removed values, there is also no much difference whether we're setting a new column of strings or ints